tests/filecheck/dialects/stencil/stencil_ops.mlir

```diff
@@ -63,17 +63,16 @@ builtin.module {
 // CHECK-NEXT:     %time_m = arith.constant 0 : index
 // CHECK-NEXT:     %time_M = arith.constant 1000 : index
 // CHECK-NEXT:     %step = arith.constant 1 : index
-// CHECK-NEXT:     %fnp1, %fn = "scf.for"(%time_m, %time_M, %step, %f0, %f1) ({
-// CHECK-NEXT:     ^0(%time : index, %fi : !stencil.field<[-4,54]x[-4,84]x[-4,44]xf32>, %fip1 : !stencil.field<[-4,54]x[-4,84]x[-4,44]xf32>):
+// CHECK-NEXT:     %fnp1, %fn = scf.for %time = %time_m to %time_M step %step iter_args(%fi = %f0, %fip1 = %f1) -> (!stencil.field<[-4,54]x[-4,84]x[-4,44]xf32>, !stencil.field<[-4,54]x[-4,84]x[-4,44]xf32>) {
```

Comment:
Do people consider this syntax better than before here?

Reply:
it's easier to read but in a WIP branch I have for this [here](https://github.com/xdslproject/xdsl/tree/christos/dialects/scf-custom-format) I follow the MLIR custom format:

```
scf.for %iv_32 = %lb_32 to %ub_32 step %step_32 : i32 {
  ... // body
}
```

which is the most unambiguous to read and understand IMHO.

What do you think @georgebisbas?

Reply:
I think it's what MLIR uses and it's the most unambiguous and easy to read/understand IMHO.